boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -52,6 +53,13 @@ document.addEventListener("DOMContentLoaded", () => {
     }
   })
 
+  document.addEventListener("keydown", (event) => {
```

Comment:
I checked the code flow, since I'm checking if the drawer is open and then if the key was `Escape` I'm invoking the `closeDrawer` function, it won't interfere with other custom elements, but if in case I continue with the modal element for event details, then I need to handle it here as well.